flask_admin/model/widgets.py

```diff
@@ -109,7 +109,10 @@ def get_kwargs(self, subfield, kwargs):
         elif subfield.type == 'BooleanField':
             kwargs['data-type'] = 'select'
             # data-source = dropdown options
-            kwargs['data-source'] = {'': 'False', '1': 'True'}
+            kwargs['data-source'] = json.dumps([
+                {'value': '0', 'text': gettext('No')},
```

Comment:
Oops, looks good then.

Any idea why it's using "0" now instead of ""? Does that cause any issues?